packages/playground/src/utils/helpers.ts

```diff
@@ -100,3 +100,25 @@ export function markAsFromAnotherClient(deployment: any): any {
   }
   return deployment;
 }
+
+export function handleKYCError(message: string, error: Error): string {
+  if (error instanceof KycErrors.TFGridKycError) {
+    switch (true) {
+      case error instanceof KycErrors.KycInvalidAddressError:
+        return `${message}: Invalid address, please contact support`;
+      case error instanceof KycErrors.KycInvalidChallengeError:
+        return `${message}: Invalid challenge, please contact support`;
```

Comment:
I don't think user would understand what 'invalid challenge' means. Can we rephrase this to perhaps 'Invalid Credentials' or 'Verification Failed'?

Reply:
As I discussed with @AhmedHanafy725, those parts of the message is for us as developers and support, we need to know what happen exactly, so we mentioned to contact support